scipio/src/timer/duration.rs

```diff
@@ -0,0 +1,324 @@
+// Unless explicitly stated otherwise all files in this repository are licensed under the
```

Comment:
I think the benefits of having this are not strong enough to outweight the confusion of users having to figure out the differences between various duration implementations.

In general we should stay away from reimplementing standard types without good reason. I'd prefer you keep everything as u64 until the very last step for presentation than doing this.

Reply:
Wiped